inductor: fix complier error when trying to vectorize logit_and and logit_or

XiaobingSuper opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #95361

Currently, `operator&& `  and `operator|| ` don't have vectorization implementation, disable them now for a quick fix for 2.0 release.

cc @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

XiaobingSuper committed:
inductor: fix comnplier error when trying to vectorize logit_and or logit_or

[ghstack-poisoned]

XiaobingSuper committed:
Update on "inductor: fix comnplier error when trying to vectorize logit_and"




cc soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

XiaobingSuper committed:
Update on "inductor: fix comnplier error when trying to vectorize logit_and"


Currently, operator&&  doesn't support vectorization, disable it now for quick fix for 2.0 release.

cc soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

XiaobingSuper commented:
@pytorchbot merge

XiaobingSuper committed:
Update on "inductor: fix comnplier error when trying to vectorize logit_and"


Currently, `operator&& ` doesn't have a vectorization implementation, disable it now for quick fix for 2.0 release.

cc soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

EikanWang commented on test/inductor/test_torchinductor.py:
```diff
@@ -5946,6 +5955,7 @@ def test_cpu_vec_cosim(self):
                 "randn",
                 "isnan",
                 "rand",
+                "logical_and",
```
Since the vec has supported logical_or, hence, I think it is better to enable logical_and in aten::vec but not just skipe it.

XiaobingSuper replied:
`logical_or` is alos not supported(https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/cpu/vec/vec512/vec512_float.h), yes, we can do it at next step.

XiaobingSuper committed:
Update on "inductor: fix complier error when trying to vectorize logit_and"


Currently, `operator&& ` doesn't have a vectorization implementation, disable it now for quick fix for 2.0 release.

cc soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

XiaobingSuper committed:
Update on "inductor: fix complier error when trying to vectorize logit_and and logit_or"


Currently, `operator&& `  and `operator|| ` don't have vectorization implementation, disable them now for a quick fix for 2.0 release.

cc soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

XiaobingSuper commented:
@pytorchbot merge
